_posts/2016-03-17-the-need-for-fit.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title:  "The need for fit"
+date:   2016-03-17 11:58:26
+author: Michael Peyton Jones
+categories:
+---
+
+Personal fit with project the is incredibly important in startup founders. 
+Startup work is demanding: it typically involves working long hours for several years,
+with the prospect of a high failure rate even if you do keep going.
```

Comment:
Why start with the motivational aspect? It might be better to start by summarising the issue of fit - are general entrepreneurial skills mostly what matters, or does fit matter?